docs/components/MessagingAttachmentView.jsx

```diff
@@ -0,0 +1,264 @@
+        onClickAttachment: () => console.log("clicked!"),
+        onRemoveAttachment: () => console.log("removed!"),
+        title: "Hover over me.m4a",
+        subtitle: "Click to download",
+        isUpload: true,
+        uploadComplete: true,
+      },
+      {
+        attachmentID: "4",
+        icon: <FileAttachmentIcon fileType={"audio"} isUpload uploadError />,
+        onClickAttachment: () => console.log("clicked!"),
+        onRemoveAttachment: () => console.log("removed!"),
+        title: "File cannot exceed 10 MB",
+        isUpload: true,
+        uploadError: true,
+      },
+    ];
+
+    return (
+      <View
+        className={cssClass.CONTAINER}
+        title="MessagingAttachment"
+        sourcePath="src/MessagingAttachment/MessagingAttachment.tsx"
+      >
+        <header className={cssClass.INTRO}>
+          <p>
+            MessagingAttachment component, to be used in AnnouncementBubble, MessagingBubble,
+            MessagingInput, etc
+          </p>
+          <p>
+            {
+              "There is a subcomponent <FileAttachmentIcon />  which takes in a fileType prop and renders an icon for for that filetype"
+            }
+          </p>
+          <CodeSample>
+            {`
+              import {MessagingAttachment} from "clever-components";
+              // OR
+              import MessagingAttachment from "clever-components/dist/MessagingAttachment"; // Avoids importing all of clever-components.
+            `}
+          </CodeSample>
+        </header>
+
+        <Example title="Basic Usage:">
+          <ExampleCode>
+            <p>Sent Attachments:</p>
+            <FlexBox className={cssClass.DEMO_ROW}>
+              {sentAttachments.map((attachment) => (
+                <MessagingAttachment
+                  attachmentID={attachment.attachmentID}
+                  icon={attachment.icon}
+                  onClickAttachment={attachment.onClickAttachment}
+                  title={attachment.title}
+                  subtitle={attachment.subtitle}
+                />
+              ))}
+            </FlexBox>
+            <p>Draft Attachments:</p>
+            <FlexBox className={cssClass.DEMO_ROW}>
+              {draftAttachments.map((attachment) => (
+                <MessagingAttachment
+                  attachmentID={attachment.attachmentID}
+                  icon={attachment.icon}
+                  onClickAttachment={attachment.onClickAttachment}
+                  onRemoveAttachment={attachment.onRemoveAttachment}
+                  title={attachment.title}
+                  subtitle={attachment.subtitle}
+                  isUpload={attachment.isUpload}
+                  uploadError={attachment.uploadError}
+                  uploadComplete={attachment.uploadComplete}
+                />
+              ))}
+            </FlexBox>
+          </ExampleCode>
+        </Example>
+
+        {this._renderProps()}
+      </View>
+    );
+  }
+
+  // TODO: Update prop documentation.
```

Comment:
Is there still something to be added to the documentation, or is this todo now completed?

Reply:
done